packages/nitro-protocol/contracts/interfaces/IForceMove.sol

```diff
@@ -1,12 +1,12 @@
-pragma solidity ^0.5.13;
+pragma solidity ^0.6.0;
 pragma experimental ABIEncoderV2;
 
 import './ForceMoveApp.sol';
 
 /**
   * @dev The IForceMove contract abstraction defines the interface that an implementation of ForceMove should implement. ForceMove protocol allows state channels to be adjudicated and finalized.
 */
-contract IForceMove {
+abstract contract IForceMove {
```

Comment:
Nice!

Reply:
Solidity 0.6.0 has stricter checks on inheritance, so this is a must now. The compiler deserves the credit 😛